pkg/cluster/docker_desktop_dial_windows.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func dockerDesktopSocketPath() (string, error) {
```

Comment:
❓ 
```
func dockerDesktopSocketPaths() ([]string, error) {
```

Reply:
oops! yep. fortunately we have some good cross-compilation tests that caught this